docs/fastdds/statistics/dds_layer/qos.rst

```diff
@@ -19,6 +19,10 @@ Statistics DataWriter recommended QoS
 -------------------------------------
 
 The following table shows the recommended |DataWriterQos-api| profile for enabling the statistics DataWriters.
+This profile enables the ``pull mode`` on the statistics DataWriters (please refer to :ref:`push_mode_qos_policy`).
```

Comment:
The link could be on "operating mode" instead of the parenthesis
```suggestion
This profile enables the ``pull mode`` operating mode on the statistics DataWriters (please refer to :ref:`push_mode_qos_policy`).
```